BetaCameras/BaslerToF/BaslerToF.cs

```diff
@@ -362,11 +364,11 @@ protected override void UpdateImpl()
                     {
                         confidenceImage = new UShortCameraImage(width, height);
 
-                        for (int y = 0; y < height; y++)
+                        for (int y = 0, i = 0; y < height; y++)
                         {
-                            for (int x = 0; x < width; x++)
+                            for (int x = 0; x < width; x++, i++)
                             {
-                                confidenceImage[y, x] = bufferConfidence[y * width + x];
+                                confidenceImage[y, x] = bufferConfidence[i++];
```

Comment:
The second i++ here (first in the loop) looks like an error.
If it's intended please add a comment.

Reply:
Corrected.

I prefer this variant, because it's more consistent: All the running variables are definied in the same place and their scope is limited to the maximum required range.

Reply:
Woo-hoo!